src/main/java/com/google/firebase/remoteconfig/Template.java

```diff
@@ -177,6 +207,25 @@ public Template setVersion(Version version) {
     return this;
   }
 
+  /**
+   * Gets the JSON-serializable representation of this template.
+   *
+   * @return A JSON-serializable representation of this {@link Template} instance.
+   */
+  public String toJSON() {
+    String jsonSerialization;
+    Gson gson = new GsonBuilder()
```

Comment:
Why not just construct a `TemplateResponse` instance here and then serialize that via `JsonFactory`? Gson shouldn't be needed.

Reply:
Good call! Updated the code to use Response types. Thanks!